### ISSUE TITLE
Sidepanels: height incorrect when using "disabledElements" option

### ISSUE THREAD

**Issue by TetsuGuy** [PDFTron/webviewer-vue-sample#32]
Hi there! First of all, thanks for providing such a great tool, i really enjoy it. 
Im building a custom UI around PDFTron with Vue.js and when disabling Elements of the WebViewer like so:

```
WebViewer(
            {
                path: "path",
                initialDoc: "url",
                disabledElements: ["header", "toolsHeader", "pageNavOverlay"]
            },
            document.getElementById("webviewer") as HTMLElement
)
```

The sidepanels don't calculate the height correctly, for example the left panel is styled like this:

```
@media (min-width: 641px)
.LeftPanel {
    height: calc(100% - 37px);
}
```
This also applies to the right panel (whose class is for whatever reason named `right-panel` with a hyphen).

The height would be correct when only disabling the `toolsHeader`, but since i also disable the `header` this should just be `100%`. 
Of course, i can add my custom stylesheet to it but since i noticed that the styling of the panel adapts to what elements are disabled, there is no need to do that.

**Comment by mparizeau-pdftron** [comment]
I believe we made a fix for this in WebViewer 7.3.3. Can you try that version out and see if it fixes things for you?

**Comment by TetsuGuy** [comment]
Thank you, the new version fixed it !
